README.md

```diff
@@ -31,16 +31,52 @@ See the [OpenAPI Schema](./schema/api.spec.yaml) for details on interacting with
 
 ## Development
 
-### Prerequisities
+### Dependencies
 
 - Golang >= 1.13
 
-### Running the service
+### Deploying locally
 
-Run `podman-compose up --build -d` to start the service and its dependencies
+Config-manager is managed by [Clowder](https://github.com/RedHatInsights/clowder) and can be deployed locally onto a [Minikube](https://minikube.sigs.k8s.io/docs/start/) instance using [Bonfire](https://github.com/RedHatInsights/bonfire).
 
-The API can be accessed locally on port 8081, and this port can be adjusted by setting the CM_WEBPORT environment variable.
+The following steps (detailed [here](https://clouddot.pages.redhat.com/docs/dev/getting-started/backend-local.html)) should be performed before attempting to build and deploy a new instance of config-manager:
 
+1. Install minikube
 ```sh
-curl -v -H "x-rh-identity: eyJpZGVudGl0eSI6eyJpbnRlcm5hbCI6eyJvcmdfaWQiOiI1MzE4MjkwIn0sImFjY291bnRfbnVtYmVyIjoiOTAxNTc4IiwidXNlciI6e30sInR5cGUiOiJVc2VyIn19Cg==" http://localhost:8081/states/current
-```
\ No newline at end of file
+$ curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-latest.x86_64.rpm
+$ sudo rpm -ivh minikube-latest.x86_64.rpm
+```
+
+2. Start minikube (check [here](https://github.com/RedHatInsights/clowder/blob/master/docs/macos.md) for MacOS instructions)
+```sh
+minikube start --cpus 4 --disk-size 36GB --memory 8000MB --addons=registry --driver=kvm2
+```
+
+3. Install Clowder CRDs
+```sh
+curl  https://raw.githubusercontent.com/RedHatInsights/clowder/master/build/kube_setup.sh -o kube_setup.sh  && chmod +x kube_setup.sh
+./kube_setup.sh
+```
+
+4. Install Clowder (replace version with [latest](https://github.com/RedHatInsights/clowder/releases/latest))
+```sh
+minikube kubectl -- apply -f https://github.com/RedHatInsights/clowder/releases/download/0.15.0/clowder-manifest-0.15.0.yaml --validate=false
+```
+
+5. Create a namespace for config-manager
+```sh
+kubectl create ns config-manager
+```
+
+6. Deploy config-manager
+```sh
+./bonfire_deploy.sh
```

Comment:
There's an additional command that needs to be run right before this. Something about creating a default env? It's the command you pasted in chat during our Google Meet.